testsuite/forge-cli/src/suites/mod.rs

```diff
@@ -1,4 +1,15 @@
 // Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
 pub mod dag;
+pub mod db;
```

Comment:
Probably for a future PR. Now that we have the test suites split up into different modules, it'd be great to have descriptions for what each test suite is doing. @igor-aptos added some comments for forge-stable in #14775 , maybe we should move some of the docs to these files?

Reply:
Yeah in general i think these should live with the tests themselves not in docs on a forge workflow